Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,11 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
```

Comment:
When would `poured_from_bottle` be neither `true` nor `false`? If that's really possible, I think it would be best to make this an `else` clause in the above `case` statement.